brainiak/utils/fmrisim.py

```diff
@@ -574,40 +576,47 @@ def double_gamma_hrf(stimfunction,
 
     """
 
+    temporal_resolution = 0.001  # What is the temporal resolution (in s)
+
     hrf_length = 30  # How long is the HRF being created
 
-    hrf = [0] * hrf_length  # How many seconds of the HRF will you model?
+    # How many seconds of the HRF will you model?
+    hrf = [0] * int(hrf_length / temporal_resolution)
 
-    for hrf_counter in list(range(0, hrf_length - 1)):
+    for hrf_counter in list(range(len(hrf) - 1)):
         # When is the peak of the two aspects of the HRF
         response_peak = response_delay * response_dispersion
```

Comment:
Hi Cameron, I think that response_delay, response_dispension and the two other terms on the right hand side of the line below do not change within the for loop. So the result of these two lines will be exactly the same in each iteration of the for loop. Therefore, it seems that they can be taken to before the for loop for slightly faster speed.